### PR TITLE
Feature/grants 27

### PR DESCRIPTION
**Pull request by zavies97**

### CODE REVIEW

File: app/config/question-bank.js [hunk 1]
```diff
@@ -993,10 +990,11 @@ const questionBank = {
               type: 'number',
               classes: 'govuk-input--width-10',
               prefix: {
-                text: '£'
+                text: '£',
+                error: 'Enter the business turnover'
```
Comment (diya2214) [inline app/config/question-bank.js:994]: @zavies97 can you explain the error here arent we handling this in validation array ??

File: app/config/question-bank.js [hunk 2]
```diff
@@ -902,16 +902,11 @@ const questionBank = {
           key: 'business-details',
           order: 180,
           title: 'Business details',
-          pageTitle: 'Crops',
+          pageTitle: '',
           url: 'business-details',
-          baseUrl: 'business-details',
+          baseurl: 'business-details',
```
Comment (diya2214) [inline app/config/question-bank.js:907]: we use camelCase **baseUrl**

Comment (zavies97) [reply app/config/question-bank.js]: This has now been updated

File: app/config/question-bank.js (hunk 1 above)
Comment (zavies97) [reply app/config/question-bank.js]: This was supposed to just be in the validation array, but I had added it into two places. Have removed the error from prefix

**Pull request merged**